components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -2053,4 +2169,38 @@ subroutine vmr2qqcw( lchnk, vmr, mbar, ncol, im, pbuf )
 
   end subroutine vmr2qqcw
 
+  !=============================================================================
+  real(r8) function air_dynamic_viscosity( temp )
+    !-----------------------------------------------------------------
+    ! Calculate dynamic viscosity of air, unit [g/cm/s]
+    !
+    ! note that this calculation is different with that used in dry deposition
+    ! see the same-name function in modal_aero_drydep.F90 
+    !-----------------------------------------------------------------
+
+    real(r8),intent(in) :: temp   ! air temperature [K]
+
+    air_dynamic_viscosity = 1.8325e-4_r8 * (416.16_r8/(temp+120._r8)) *    &
+                            ((temp/296.16_r8)**1.5_r8)
+
+  end function air_dynamic_viscosity
+
+  !=============================================================================
+  real(r8) function air_kinematic_viscosity( temp, rhoair )
+    !-----------------------------------------------------------------
+    ! Calculate kinematic viscosity of air, unit [cm^2/s]
+    !-----------------------------------------------------------------
+
+    real(r8),intent(in) :: temp     ! air temperature [K]
+    real(r8),intent(in) :: rhoair   ! air density [g/cm2]
```

Comment:
The units for air density should be [g/cm^3]

Reply:
corrected, Thank you.